bin/desi_compute_pixel_flatfield

```diff
@@ -236,21 +236,21 @@ def amplifier_matching(image) :
     vband_a = np.median(image[c0-lw:c0,c1-w:c1],axis=1)
     vband_b = np.median(image[c0-lw:c0,c1::c1+w],axis=1)
     rab = np.sqrt( np.median(vband_a/vband_b) / np.median(vband_b/vband_a) ) # to debias
-    log.debug("a/b=",rab)
+    log.debug("a/b={}".format(rab))
```

Comment:
In this case you are fixing a logging bug, and this statement isn't buried in a loop so performance doesn't matter much, but in general we should get in the habit of formatting log.debug statements with
```
log.debug("a/b=%f", rab)
```
The difference is that if the logger knows it won't print the message, it skips evaluating the format string, whereas the way it is currently written the string is evaluated whether or not it will be printed.  In the past that has mattered when we accidentally had a formatted debug string inside a nested loop and ended up taking a lot of time just on formatting a debug statement that was normally never printed.